README.md

```diff
@@ -74,6 +86,43 @@ If you work with [Q-cli](https://github.com/nzzdev/Q-cli) there is a config file
 
 Of course with another mapbox configuration you have to adjust the LAYER_CONFIGS defintion as well so that it fits your needs.
 
+[to the top](#table-of-contents)
+
+## Development
+
+Install the [Q cli](https://github.com/nzzdev/Q-cli) and start the Q dev server:
+
+```
+$ Q server
+```
+
+Run the Q tool:
+```
+$ npm run index.js
```

Comment:
`$ node dev.js`

Reply:
Fixed in https://github.com/nzzdev/Q-map/pull/86/commits/339516aa2839816ac1ad5703c7216a16e9294ca9